### ISSUE TITLE
iOS accessibility: menu button has border when not visible

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter#4752]
Turn on voice over in iOS. Open gallery. Tap the menu icon. Then double tap.

Expected: the border around menu button disappears once the side drawer is fully opened (to see this in action, notice what Inbox does).

Actual:

![1a351d5d-6425-44a9-81f5-9ebc7351e413](https://cloud.githubusercontent.com/assets/5479/16352956/251cbb2c-3a28-11e6-9683-b9f540bfb804.PNG)

**Comment by goderbauer** [comment]
This has been fixed.
